submissions/smillims/js-dom/script.js

```diff
@@ -29,21 +29,18 @@ const nav = document.querySelector('.navbar');
 const ul = document.querySelector('.list');
 const container = document.querySelector('.container');
 
-function createNavItem() {
-	let content = '';
-	heroes.map(({id, name}) => {
-		content += `<li class='list-item'>
-		<a data-id="${id}" class="list-link" href="#">${name}</a>
+const createNavItem = () => {
+	ul.innerHTML = heroes.map(({id, name}) => (
+		`<li class='list-item'>
+			<a data-id="${id}" class="list-link" href="#">${name}</a>
 		</li>`
-	})
-	
-	ul.innerHTML = content;
+	)).join('')
 }
 
 function displayFact(navLink) {
 	const hero = heroes.find((el) => navLink.dataset.id == el.id)
  	if (!hero) {
- 		return;
+ 		return hero = '';
```

Comment:
What you want to do here ?

p.s.
```
Uncaught TypeError: Assignment to constant variable.
    at displayFact (script.js:43)
    at HTMLUListElement.<anonymous> (script.js:63)
```

Reply:
wanted to clean line :disappointed_relieved:

Reply:
okay, I was deleted this statement and everything work ^^